# ダイ温度(旧ロボットベース、SCIBOT-Xのみ)SET
float64 value
---

